### PR TITLE
new put api added

### PR DESCRIPTION
**Pull request by GANGSTER0910**
In this, I have added the put api for the post request of analyze_img_file here first we have to add the data through post api and then with the help of put api we can change the orignal data and we can also see the changes of old and new file #16

### CODE REVIEW

**Comment by ashitaprasad** [comment]
Can you create a different POST API which is simpler (text bases) and add PUT API for it.

**Comment by GANGSTER0910** [comment]
> Can you create a different POST API which is simpler (text bases) and add PUT API for it.

I have done the work you have asked, so please review it once more

File: src/routes/io/io.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import io
 import time
 import puremagic
+from pydantic import BaseModel
```
Comment (ashitaprasad) [inline src/routes/io/io.py:4]: Why are you importing this?

Comment (GANGSTER0910) [reply src/routes/io/io.py]: Sorry, It was imported by mistake, I have resolve the issue

File: src/routes/io/io.py [hunk 2]
```diff
@@ -80,3 +79,45 @@ async def analyze_img_file(
...
+            "password": password
+        }
+        return ok_200({"username":username,
+                       "email" : email,
+                       "password": password})
+    except:
+        raise internal_error_500()
+
+@io_router.put('/user')
+async def create_user(old_username:str
```
Comment (ashitaprasad) [inline src/routes/io/io.py:103]: Why is the function having same name ?

File: src/routes/io/io.py [hunk 3]
```diff
@@ -80,3 +79,45 @@ async def analyze_img_file(
...
+,    old_email:str
+,    old_password:str, new_username:str
```
Comment (ashitaprasad) [inline src/routes/io/io.py:105]: What is the use of old_email & old_password if it is not being used anywhere

File: src/routes/io/io.py (hunk 2 above)
Comment (ashitaprasad) [inline src/routes/io/io.py:103]: Also username is unique, it should not change. There is no use of old_username and new_username.

Comment (GANGSTER0910) [reply src/routes/io/io.py:103]: I should have name it update user

File: src/routes/io/io.py (hunk 3 above)
Comment (GANGSTER0910) [reply src/routes/io/io.py:105]: old email means email which is first created by post request and if wants to change then he can change it with new email

File: src/routes/io/io.py (hunk 2 above)
Comment (GANGSTER0910) [reply src/routes/io/io.py:103]: ok I will give functionality only to email and password

**Comment by GANGSTER0910** [comment]
I have done the changes you have asked, sorry for above code and please review it

File: src/routes/io/io.py [hunk 4]
```diff
@@ -80,3 +79,44 @@ async def analyze_img_file(
...
+
+@io_router.put('/user/update')
+async def update_user(username:str
+,    email:str
+,    password:str
+,    new_email:str
+,    new_password:str):
+    try:
+        old_file_data = file_data[username]
+        new_file_data= {
```
Comment (ashitaprasad) [inline src/routes/io/io.py:110]: new_file_data should be such that:
- if new_email is empty or None then the existing will be used.
- if new_password is empty or None then the existing will be used.

File: src/routes/io/io.py [hunk 5]
```diff
@@ -80,3 +79,44 @@ async def analyze_img_file(
...
+        return ok_200({"username":username,
+                       "email" : email,
+                       "password": password})
+    except:
+        raise internal_error_500()
+
+@io_router.put('/user/update')
+async def update_user(username:str
+,    email:str
+,    password:str
```
Comment (ashitaprasad) [inline src/routes/io/io.py:105]: remove email & password from here as it is never used inside.

**Comment by GANGSTER0910** [comment]
The second file was added by mistake so ignore it, I have also removed the changes from commit

File: src/routes/text/case_convert.py
Comment (ashitaprasad) [inline src/routes/text/case_convert.py:1]: Why have you deleted this file.

**Comment by ashitaprasad** [comment]
Merging this change will delete `case_convert.py` from main repo. 
Make a fresh branch .. make only changes in the required file and raise a new PR.

**Pull request closed without merging**